tests/Unit/Visualization/Python/Test_PlotTrajectories.py

```diff
@@ -0,0 +1,143 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import shutil
+import unittest
+
+import numpy as np
+from click.testing import CliRunner
+
+import spectre.IO.H5 as spectre_h5
+from spectre.Informer import unit_test_build_path, unit_test_src_path
+from spectre.Visualization.PlotTrajectories import plot_trajectories_command
+
+
+class TestPlotTrajectories(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = os.path.join(
+            unit_test_build_path(), "Visualization", "PlotTrajectories"
+        )
+        self.h5_filename = os.path.join(
+            unit_test_src_path(),
+            "Visualization/Python",
+            "TestPlotTrajectoriesReductions.h5",
+        )
+        os.makedirs(self.test_dir, exist_ok=True)
+        self.create_h5_file_if_not_exists()
+
+    def tearDown(self):
+        if TestPlotTrajectories.delete_test_dir:
+            shutil.rmtree(self.test_dir)
+
+    # if h5 test file does not exist then create one using mock-up inspiral data
+    def create_h5_file_if_not_exists(self):
+        """Create the H5 file if it does not exist."""
+        if not os.path.exists(self.h5_filename):
```

Comment:
You can delete this `if`. Instead, put `shutil.rmtree(self.work_dir, ignore_errors=True)` just before `os.makedirs` in the `setUp` function above. Then the test always starts from a clean slate.